CHANGELOG.md

```diff
@@ -3,7 +3,14 @@
 ## Fixes
 
 - [Pull request #1104: Change visually hidden footer title](https://github.com/alphagov/govuk-prototype-kit/pull/1104)
-- [Pull request #1076: Update to Node.js v14.17.6 when using Node Version Manager](https://github.com/alphagov/govuk-prototype-kit/pull/1076)
+
+### Updating Node.js
+
+We are recommending that all users of the GOV.UK Prototype Kit update to the latest version of Node.js 14 LTS, to make sure they are protected from a [recent security vulnerability in the Node package manager npm](https://github.blog/2021-09-08-github-security-update-vulnerabilities-tar-npmcli-arborist/).
```

Comment:
```suggestion
If you use the GOV.UK Prototype Kit, then we recommend you update to the latest version of Node.js 14 Long Term Support (LTS).

This is to make sure you're protected from a recent security vulnerability in the Node Package Manager (npm).
```

Reply:
Thanks, this reads much better! It does remove the link though, is that on purpose?

Reply:
No, that's my mistake! Ta for flagging. Will add back in.

```suggestion
If you use the GOV.UK Prototype Kit, then we recommend you update to the latest version of Node.js 14 Long Term Support (LTS).

This is to make sure you're protected from a [recent security vulnerability in the npm (Node Package Manager)](https://github.blog/2021-09-08-github-security-update-vulnerabilities-tar-npmcli-arborist/).
```